pipelines/nextflow/modules/genome_metadata/prepare_genome_metadata.nf

```diff
@@ -14,23 +14,18 @@
 // limitations under the License.
 
 process PREPARE_GENOME_METADATA {
+    tag "$accession"
     label 'local'
 
     input:
-        path(input_json, stageAs: "input_genome.json")
+        tuple val(accession), path(input_json), path(ncbi_json)
 
     output:
-        path ("genome.json"), emit: genomic_dataset
+        tuple val(accession), path("genome.json"), emit: genomic_dataset
         
     shell:
     output_json = "genome.json"
     '''
-    genome_metadata_prepare --input_file !{input_json} --output_file !{output_json}
+    genome_metadata_prepare --input_file !{input_json} --output_file !{output_json} --ncbi_meta !{ncbi_json}
```

Comment:
Why have you removed the stub?

Reply:
This part no longer needs to be mocked: the ncbi_meta generated by the datasets module already produces the mock file in its own stub